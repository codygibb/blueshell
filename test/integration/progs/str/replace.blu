print "Hello x, your name is x".replace("x", "Cody")
print "a.b.cc.b.a".replace("[a-c]", "-")
print "nothing to replace".replace("[x-z]+", "?")
