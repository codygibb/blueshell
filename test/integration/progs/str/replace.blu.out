Hello Cody, your name is Cody
-.-.--.-.-
nothing to replace
